xayn-ai/src/coi/system.rs

```diff
@@ -159,6 +160,37 @@ impl CoiSystem {
             neg_distance,
         })
     }
+
+    /// Computes the relevance/weights of the cois.
+    ///
+    /// The weights are computed from the view counts and view times of each coi and they are not
+    /// normalized. The horizon specifies the time since the last view after which a coi becomes
+    /// irrelevant.
+    #[allow(dead_code)]
+    fn compute_weights<CP: CoiPoint>(&self, cois: &[CP], horizon: Duration) -> Vec<f32> {
+        let counts = cois.iter().map(|coi| coi.view().count).sum::<usize>() as f32 + f32::EPSILON;
+        let times = cois
+            .iter()
+            .map(|coi| coi.view().time)
+            .sum::<Duration>()
+            .as_secs_f32()
+            + f32::EPSILON;
+        let now = system_time_now();
+        let horizon = (-0.1 * horizon.as_secs_f32() / SECONDS_PER_DAY).exp();
```

Comment:
Can we give a name to `-0.1`?

Reply:
in theory yes, but i wouldn't know which name? it's just a scaling constant without any special meaning. i also wouldn't give a name to `3` in `let y = 3 * x;` for example.

Reply:
I'm also not sure how to call it. `horizon_scale` can work for the moment. we are using it in two points so it's better to have a variable for it. In the future I guess this will be a configuration of the ranker